cmd/oci-image-tool/create.go

```diff
@@ -40,9 +41,10 @@ func createHandle(context *cli.Context) error {
 	}
 
 	v := bundleCmd{
-		typ:  context.String("type"),
-		ref:  context.String("ref"),
-		root: context.String("rootfs"),
+		typ:      context.String("type"),
+		ref:      context.String("ref"),
+		root:     context.String("rootfs"),
+		platform: context.StringSlice("platform"),
```

Comment:
As I said above, when you find and validate multiple `manifest` in `index`, the following create and unpack functions don't know which `manifest` is called specifically. I added this flag to add input parameters to determine the specific `manifest`.